src/queries/queries.ts

```diff
@@ -6,7 +6,12 @@ import { locationToView } from '../model';
 export const currentView = getCurrentView(locationToView);
 
 export const restaurants = queryShallow(
-  (parms: { searchQuery: string; locationQuery: string; radiusQuery: number }) =>
-    API.getRestaurants(parms.searchQuery, parms.locationQuery, parms.radiusQuery),
+  (params: { searchQuery: string; locationQuery: string; radiusQuery: number }) =>
+    API.getRestaurants(params.searchQuery, params.locationQuery, params.radiusQuery),
+  expire(10000)
+);
+
+export const business = queryShallow(
+  (params: { businessId: string }) => API.getBusinessDetails(params.businessId),
```

Comment:
in questo caso, visto che l'input è una semplice stringa, basterebbe anche
```suggestion
export const business = queryStrict(
  API.getBusinessDetails,
```
comunque cambia poco